magefiles/test.go

```diff
@@ -180,6 +180,11 @@ func patchAddonOperatorCSV(ctx context.Context,
 						Name:  "ENABLE_STATUS_REPORTING",
 						Value: "true"},
 					)
+					// Set Upgrade policy status reporting env variable to true.
+					containerObj.Env = append(containerObj.Env, corev1.EnvVar{
+						Name:  "ENABLE_UPGRADEPOLICY_STATUS",
+						Value: "true"},
+					)
```

Comment:
@supreeth7  this mage function has that change right . Like based on your previous comment

Reply:
You'll need to inject it somewhere here: https://github.com/openshift/addon-operator/blob/9d74480a5687977a0f978788402b33a4027f9492/magefiles/test.go#L113

Reply:
@supreeth7  actually IntegrationCIInjectEnvVariable calls injectStatusReportingEnvironmentVariable and it calls patchAddonOperatorCSV where this change is already made .

Reply:
Ah, missed it :+1: